src/scene/materials/standard-material.js

```diff
@@ -447,18 +447,43 @@ let _params = new Set();
  * @property {string} lightVertexColorChannel Vertex color channels to use for baked lighting. Can
  * be "r", "g", "b", "a", "rgb" or any swizzled combination.
  * @property {boolean} ambientTint Enables scene ambient multiplication by material ambient color.
- * @property {import('../../platform/graphics/texture.js').Texture|null} aoMap Baked ambient
+ * @property {import('../../platform/graphics/texture.js').Texture|null} aoMap The main (primary) baked ambient
  * occlusion (AO) map (default is null). Modulates ambient color.
- * @property {number} aoMapUv AO map UV channel
- * @property {string} aoMapChannel Color channel of the AO map to use. Can be "r", "g", "b" or "a".
- * @property {Vec2} aoMapTiling Controls the 2D tiling of the AO map.
- * @property {Vec2} aoMapOffset Controls the 2D offset of the AO map. Each component is between 0
+ * @property {number} aoMapUv Main (primary) AO map UV channel
+ * @property {string} aoMapChannel Color channel of the main (primary) AO map to use. Can be "r", "g", "b" or "a".
+ * @property {Vec2} aoMapTiling Controls the 2D tiling of the main (primary) AO map.
+ * @property {Vec2} aoMapOffset Controls the 2D offset of the main (primary) AO map. Each component is between 0
  * and 1.
- * @property {number} aoMapRotation Controls the 2D rotation (in degrees) of the AO map.
+ * @property {number} aoMapRotation Controls the 2D rotation (in degrees) of the main (primary) AO map.
  * @property {boolean} aoVertexColor Use mesh vertex colors for AO. If aoMap is set, it'll be
  * multiplied by vertex colors.
  * @property {string} aoVertexColorChannel Vertex color channels to use for AO. Can be "r", "g",
  * "b" or "a".
+ * @property {import('../../platform/graphics/texture.js').Texture|null} aoDetailMap The
+ * detail (secondary) baked ambient occlusion (AO) map of the material (default is null). Will only be used if main
+ * (primary) ao map is non-null.
+ * @property {number} aoDetailMapUv Detail (secondary) AO map UV channel.
+ * @property {Vec2} aoDetailMapTiling Controls the 2D tiling of the detail (secondary) AO
+ * map.
+ * @property {Vec2} aoDetailMapOffset Controls the 2D offset of the detail (secondary) AO
+ * map. Each component is between 0 and 1.
+ * @property {number} aoDetailMapRotation Controls the 2D rotation (in degrees) of the detail
+ * (secondary) AO map.
+ * @property {string} aoDetailMapChannel Color channels of the detail (secondary) AO map
+ * to use. Can be "r", "g", "b" or "a".
```

Comment:
This should mention the default value is 'g'

Reply:
Is it common to use default value 'g' instead or 'r' in other similar properties?

Reply:
We don't seem to have that mentioned elsewhere in the docs, but we do set 'g' as the default channel in the material. It should be down to how textures are packed, surely, unless there is some bit depth favor for green channels (which would make sense perceptually) for certain file formats?

Reply:
I've added a clarification about the default value. I can't figure out why 'g' is the default for ao either, but I figured I'd just follow the same standard as  `aoMapChannel`. 
When packing ORM-textures for glTF, "r" is used for occlusion. But it may be related to inherent precision issues as @GSterbrant mentions.